lua/ulx_yeet/sh_yeet.lua

```diff
@@ -0,0 +1,173 @@
+if SERVER then
+    ULib.ucl.registerAccess( "physgunragdollplayer", ULib.ACCESS_ADMIN, "Ability to physgun ragdoll other players.", "Other" )
+end
+
+CreateConVar( "ulx_physgun_ragdoll_velocity", 150, { FCVAR_REPLICATED, FCVAR_ARCHIVE }, "The velocity required for a physgunned player to turn into a ragdoll on release.", 0 )
+
+local function savePlayer( ply )
+    local result = {
+        health = ply:Health(),
+        armor = ply:Armor()
+    }
+
+    if ply:GetActiveWeapon():IsValid() then
+        result.currentWeapon = ply:GetActiveWeapon():GetClass()
+    end
+
+    local weapons = ply:GetWeapons()
+
+    for _, weapon in ipairs( weapons ) do
+        result.weapondata = {}
+        printname = weapon:GetClass()
+        result.weapondata[ printname ] = {
```

Comment:
Probably don't want to set `weapondata = {}` every iteration of this loop. Also, `printname` was a global var

```suggestion
        local className = weapon:GetClass()
        result.weaponData[className] = {
```